webrender_traits/src/display_item.rs

```diff
@@ -49,20 +49,20 @@ impl BorderRadius {
         }
     }
 
-    pub fn is_uniform(&self) -> Option<f32> {
-        let uniform_radius = LayoutSize::new(self.top_left.width, self.top_left.width);
+    pub fn is_uniform(&self) -> Option<LayoutSize> {
+        let uniform_radius = LayoutSize::new(self.top_left.width, self.top_left.height);
```

Comment:
nit: Could this just be `let uniform_radius = self.top_left;`? Or just compare with `self.top_left` in the if statement. If you prefer this way, thats fine.